receiver/azureblobrehydrationreceiver/config.go

```diff
@@ -0,0 +1,101 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azureblobrehydrationreceiver //import "github.com/observiq/bindplane-agent/receiver/azureblobrehydrationreceiver"
+
+import (
+	"errors" // timeFormat is the format for the starting and end time
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+)
+
+const timeFormat = "2006-01-02T15:04"
```

Comment:
Did you mean to have the comment above the const?

```suggestion
	"errors"
	"fmt"
	"time"

	"go.opentelemetry.io/collector/component"
)

// timeFormat is the format for the starting and end time
const timeFormat = "2006-01-02T15:04"
```

Reply:
Yeah not sure what happened there good catch.